fhireval/data/CMG-ExampleData.json

```diff
@@ -0,0 +1,8486 @@
+{
```

Comment:
Most of the resources are repeating and it seems too many for testing?

Reply:
There are variations in there to suite different queries and filters. I'm sure there is some amount of data that could be eliminated but does it really matter?